receiver/receivercreator/README.md

```diff
@@ -56,6 +56,34 @@ config:
    endpoint: `endpoint`:8080
 ```
 
+**receivers.&lt;receiver_type/id&gt;.resource_attributes**
+
+This setting controls what resource attributes are set on metrics emitted from the created receiver. These attributes can be set from [values in the endpoint](#rule-expressions) that was matched by the `rule`. These attributes vary based on the endpoint type. These defaults can be disabled by setting the attribute to be removed to an empty value. Note that the values can be dynamic and processed the same as in `config`.
```

Comment:
>These defaults can be disabled by setting the attribute to be removed to an empty value.

Perhaps find a nicer way to disable the defaults. Technically, empty value is a valid attribute value.

Reply:
Option 1.

Have a `enable_default_resource_attributes: true|false` (default true) for whether the defaults are included or not. If a user just wanted some of the defaults they would have to turn the defaults off then add the ones they want to the `resource_attributes` config.

Option 2.

It seems uncommon to want to set a null or empty attribute here. For these edge cases a user could use attribute processor instead.

@tigrannajaryan I'm leaning towards the more user-friendly one which is Option 1. I think users wanting to use null or empty attribute values seems pretty unlikely and in that case they could use a processor to set them if they really wanted to. Any other ideas?

Reply:
@jrcamp Option 2 SGTM, just make the docs clear.